crates/daphne-server/src/router/leader.rs

```diff
@@ -56,12 +57,45 @@ async fn upload<A>(
 where
     A: DapLeader + DaphneService + Send + Sync,
 {
+    if req.version == DapVersion::Draft09 {
```

Comment:
Defensive programming
```suggestion
    if req.version != DapVersion::DraftLatest {
```

Reply:
So I was trying to be defensive, by saying that Draft09 is frozen, whereas DraftLatest will change.

Reply:
Excellent point, I think you're right.